<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>535</height>
   </rect>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>411</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="Button_Clear_entry">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Clear">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_delete">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_divide">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>➗</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_multiply">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>✖</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_plus_minus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>±</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_0">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_decimal">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bodoni MT Condensed</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_plus">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_equality">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>300</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＝</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_minus">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>➖</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_reciprocal">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1/x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_root">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>√</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_mod">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_square">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x²</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="Button_pai">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Π</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_factorial">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>n!</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_Left_parenthesis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>（</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_pow">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>pow</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Exp">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_sin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_10x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10ˆX</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_Right_parenthesis">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_log">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_tan">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_cos">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_square_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x²</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_root_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>√</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_mod_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>mod</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_reciprocal_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1/x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Clear_entry_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Clear_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_delete_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_divide_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>➗</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_plus_minus_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>±</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_07">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_04">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_01">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_08">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_05">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_02">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_00">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_09">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_06">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_03">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_decimal_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bodoni MT Condensed</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_multiply_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>✖</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_minus_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>➖</string>
     </property>
    </widget>
    <widget class="MyButton" name="Button_plus_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_equality_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＝</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>中南大学计科2202陈康坤计算器</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>396</width>
     <height>115</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>8</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
